metpy/units.py

```diff
@@ -57,7 +57,7 @@ def pandas_dataframe_to_unit_arrays(df, column_units=None):
 
     Parameters
     ----------
-    df : `pandas.DataFrame`
+    df : `pandas.DataFrame()`
```

Comment:
I'm surprised the parenthesis are required. Any chance that requirement goes away with `:class:`?

Reply:
There are weird things here with how it checks it. For some reason this works but `:class:` doesn't, but nothing works for the `numpy.ma.MaskedArray` call later in the same file.